crates/red_knot_python_semantic/src/types/slots.rs

```diff
@@ -24,7 +24,8 @@ enum SlotsKind {
 
 impl SlotsKind {
     fn from(db: &dyn Db, base: ClassLiteralType) -> Self {
-        let Symbol::Type(slots_ty, bound) = base.own_class_member(db, "__slots__").symbol else {
+        let Symbol::Type(slots_ty, bound) = base.own_class_member(db, None, "__slots__").symbol
```

Comment:
Not completely sure about the `None` here...

Reply:
That's correct, since `base` is a class literal (so if it's a generic class, it's not specialized yet), so it can't possibly have a specialization to pass down